docs/user/subprojects.rst

```diff
@@ -1,43 +1,67 @@
 
-For example, a parent project, ``Foo`` is set up with a subproject, ``Bar``. The
-documentation for ``Foo`` will be available at:
+Read the Docs projects can be configured to make other *subprojects* available on the website of the *main project*.
+This allows for documentation projects to share a search index and a namespace or custom domain,
+but still be maintained independently.
 
-https://foo.readthedocs.io/en/latest/
+This is useful for:
 
-The documentation for ``Bar`` will be available under this same path:
+* Organizations that need all their projects visible in one documentation portal or landing page
+* Projects that document and release several packages or extensions
+* Organizations or projects that want to have a common search function for several sets of documentation
 
-https://foo.readthedocs.io/projects/bar/en/latest/
+For a main project ``example-project``, a subproject ``example-project-plugin`` can be made available as follows:
 
-Adding a subproject
--------------------
+* Main project: https://example-project.readthedocs.io/en/latest/
+* Subproject: https://example-project.readthedocs.io/projects/plugin/en/latest/
 
-In the admin dashboard for your project, select "Subprojects" from the menu.
-From this page you can add a subproject by typing in the project slug.
+.. seealso::
 
-Subproject aliases
-~~~~~~~~~~~~~~~~~~
+   :doc:`/guides/subprojects`
+     Learn how to create and manage subprojects
+
+   :doc:`/guides/intersphinx`
+     Learn how to use references between different Sphinx projects, for instance between subprojects
 
-You can use an alias for the subproject when it is created. This allows you to override the URL that is used to access it, giving more configurability to how you want to structure your projects.
 
 Sharing a custom domain
 -----------------------
 
-Projects and subprojects can also be used to share a custom domain with a number
-of projects. To configure this, one project should be established as the parent
-project. This project will be configured with a custom domain. Projects can then
-be added as subprojects to this parent project.
+Projects and subprojects can be used to share a custom domain.
+To configure this, one project should be established as the main project.
+This project will be configured with a custom domain.
+Other projects are then added as subprojects to the main project.
+
+If the example project ``example-project`` was set up with a custom domain,
+``docs.example.com``, the URLs for projects ``example-project`` and ``example-project-plugin`` with alias ``plugin`` would
+respectively be at:
+
+* ``example-project``: https://docs.example.com/en/latest/
+* ``example-project-plugin``: https://docs.example.com/projects/plugin/en/latest/
+
+Using aliases
+-------------
+
+You can choose an alias for the subproject when it is created.
+This allows you to override the URL that is used to access it,
+giving more control over how you want to structure your projects.
```

Comment:
I'd invert this to mention the benefit first.

```suggestion
Adding an alias for the subproject allows you to override the URL that is used to access it,
giving more control over how you want to structure your projects.
You can choose an alias for the subproject when it is created.
```

Reply:
The marketing discussion is seeping in :D

Reply:
🧑‍🎓